MavenCentral username and password properties no longer work

The README says to use `mavenCentralRepositoryUsername` and `mavenCentralRepositoryPassword` but that fails when running the publish task. It says that `mavenCentralUsername` and `mavenCentralPassword` are missing. Renaming the properties to drop `Repository` works.

I got this error now when running `closeAndReleaseRepository`. Is that related to this issue?

```
> Task :closeAndReleaseRepository FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Some problems were found with the configuration of task ':closeAndReleaseRepository' (type 'CloseAndReleaseRepositoryTask').
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.repositoryPassword' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'nexusOptions.repositoryPassword'.
      2. Mark property 'nexusOptions.repositoryPassword' as optional.
    
1 actionable task: 1 executed
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.repositoryUsername' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'nexusOptions.repositoryUsername'.
      2. Mark property 'nexusOptions.repositoryUsername' as optional.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.stagingProfile' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'nexusOptions.stagingProfile'.
      2. Mark property 'nexusOptions.stagingProfile' as optional.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
```

I guess it is, because adding `mavenCentralRepositoryUsername` and `mavenCentralRepositoryPassword` gives me a different error:

```
> Task :closeAndReleaseRepository FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':closeAndReleaseRepository' (type 'CloseAndReleaseRepositoryTask').
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.stagingProfile' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'nexusOptions.stagingProfile'.
      2. Mark property 'nexusOptions.stagingProfile' as optional.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
```

--- a/README.md
+++ b/README.md
@@ -78,9 +78,9 @@ allprojects {
 }
 ```
 
-The username and password for Sonatype OSS can be provided as Gradle properties called `mavenCentralRepositoryUsername` 
-and `mavenCentralRepositoryPassword` to avoid having to commit them. You can also supply them as environment variables
-called `ORG_GRADLE_PROJECT_mavenCentralRepositoryUsername` and `ORG_GRADLE_PROJECT_mavenCentralRepositoryPassword`.
+The username and password for Sonatype OSS can be provided as Gradle properties called `mavenCentralUsername`
+and `mavenCentralPassword` to avoid having to commit them. You can also supply them as environment variables
+called `ORG_GRADLE_PROJECT_mavenCentralUsername` and `ORG_GRADLE_PROJECT_mavenCentralPassword`.
 
 
 You can add additional repositories to publish to using the standard Gradle APIs:

--- a/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishBaseExtension.kt
+++ b/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishBaseExtension.kt
@@ -30,7 +30,7 @@ abstract class MavenPublishBaseExtension(
    * repository instead.
    *
    * This expects you provide your Sonatype user name and password through Gradle properties called
-   * `mavenCentralRepositoryUsername` and `mavenCentralRepositoryPassword`.
+   * `mavenCentralUsername` and `mavenCentralPassword`.
    *
    * The `closeAndReleaseRepository` task is automatically configured for Sonatype OSSRH using the same credentials.
    *
@@ -73,8 +73,8 @@ abstract class MavenPublishBaseExtension(
 
     nexusOptions {
       it.baseUrl = "${host.rootUrl}/service/local/"
-      it.repositoryUsername = project.findOptionalProperty("mavenCentralRepositoryUsername")
-      it.repositoryPassword = project.findOptionalProperty("mavenCentralRepositoryPassword")
+      it.repositoryUsername = project.findOptionalProperty("mavenCentralUsername")
+      it.repositoryPassword = project.findOptionalProperty("mavenCentralPassword")
     }
   }
 

--- a/src/main/kotlin/com/vanniktech/maven/publish/legacy/Properties.kt
+++ b/src/main/kotlin/com/vanniktech/maven/publish/legacy/Properties.kt
@@ -27,17 +27,17 @@ internal fun Project.checkProperties() {
       "https://docs.gradle.org/current/userguide/publishing_maven.html#publishing_maven:repositories"
   }
   check(findOptionalProperty("SONATYPE_NEXUS_USERNAME") == null) {
-    "The property SONATYPE_NEXUS_USERNAME was removed. Use mavenCentralRepositoryUsername instead."
+    "The property SONATYPE_NEXUS_USERNAME was removed. Use mavenCentralUsername instead."
   }
   check(System.getenv("SONATYPE_NEXUS_USERNAME") == null) {
     "The env var SONATYPE_NEXUS_USERNAME was removed. Use the Gradle property " +
-      "mavenCentralRepositoryUsername or the env var ORG_GRADLE_PROJECT_mavenCentralRepositoryUsername instead."
+      "mavenCentralUsername or the env var ORG_GRADLE_PROJECT_mavenCentralUsername instead."
   }
   check(findOptionalProperty("SONATYPE_NEXUS_PASSWORD") == null) {
-    "The property SONATYPE_NEXUS_PASSWORD was removed. Use mavenCentralRepositoryPassword instead."
+    "The property SONATYPE_NEXUS_PASSWORD was removed. Use mavenCentralPassword instead."
   }
   check(System.getenv("SONATYPE_NEXUS_PASSWORD") == null) {
     "The env var SONATYPE_NEXUS_PASSWORD was removed. Use the Gradle property " +
-      "mavenCentralRepositoryPassword or the env var ORG_GRADLE_PROJECT_mavenCentralRepositoryPassword instead."
+      "mavenCentralPassword or the env var ORG_GRADLE_PROJECT_mavenCentralPassword instead."
   }
 }

